### PR TITLE
Two way radio

### PR DESCRIPTION
**Pull request by ajayyy**

### CODE REVIEW

File: projects/HotFire/config/UOStateMachine.cpp [hunk 1]
```diff
@@ -35,6 +36,24 @@ void UOStateMachine::StartFillingEXT()
...
+{
+	BEGIN_TRANSITION_MAP					   // - Current State -
+		TRANSITION_MAP_ENTRY(EVENT_IGNORED)	   // ST_INIT
+		TRANSITION_MAP_ENTRY(EVENT_IGNORED)	   // ST_WAIT_FOR_INIT
+		TRANSITION_MAP_ENTRY(ST_ABORT_FILLING) // ST_WAIT_FOR_FILLING
+		TRANSITION_MAP_ENTRY(ST_ABORT_FILLING) // ST_FILLING
+		TRANSITION_MAP_ENTRY(ST_ABORT_FILLING) // ST_WAIT_FOR_IGNITION
+		TRANSITION_MAP_ENTRY(ST_ABORT_BURN)	   // ST_IGNITION
+		TRANSITION_MAP_ENTRY(ST_ABORT_BURN)	   // ST_FULL_BURN
+		TRANSITION_MAP_ENTRY(EVENT_IGNORED)	   // ST_FINAL_VENTING
```
Comment (ajayyy) [inline projects/HotFire/config/UOStateMachine.cpp:50]: Do we want an abort for `ST_FINAL_VENTING`

Comment (sebastiengrd) [reply projects/HotFire/config/UOStateMachine.cpp:50]: What state would that transition to?

Comment (ajayyy) [reply projects/HotFire/config/UOStateMachine.cpp:50]: Not sure. I guess this state is already an "abort" state, just a successful one, so it wouldn't make sense to abort from that.

**Comment by ajayyy** [comment]
To be tested with https://github.com/uorocketry/RocketGroundStation2020/releases/tag/3.0

**Comment by sebastiengrd** [comment]
It was tested with the ground station and it is working

**Review by sebastiengrd — APPROVED** [review]

**Pull request merged**